utils/docker/Dockerfile.centos:7

```diff
@@ -49,8 +49,8 @@ RUN yum install -y python34-PyYAML.x86_64 PyYAML.x86_64
 RUN yum install -y python2-paramiko python34-paramiko
 RUN yum install -y valgrind rsync
 RUN yum install -y ShellCheck
-RUN yum install -y python-jwt.noarch
 RUN yum --enablerepo=epel-testing -y install python2-pygithub.noarch
+RUN yum install -y python-jwt.noarch
```

Comment:
You shouldn't need to use `epel-testing` for these any more and should be able to just do

```
RUN yum -y install python2-pygithub.noarch
```
now.